vignettes/trial_analysis.Rmd

```diff
@@ -0,0 +1,606 @@
+---
+title: "Trial Analysis"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Trial Analysis}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(crmPack)
+library(tidyverse)
+library(knitr)
+library(kableExtra)
+```
+## Introduction
+This vignette picks up where the previous one ([Trial Definition](trial_definition.html)), ends.  To recap, our trial defines the six fundamental elements of a CRM trial as 
```

Comment:
hm can we avoid the duplication somehow? e.g. really just have the one code chunk defining the trial and then continue. this will simplify maintenance etc.